inspire_disambiguation/api.py

```diff
@@ -151,6 +176,15 @@ def cluster(ethnicity_model_path, distance_model_path, n_jobs, signature_block=N
     clusterer.fit(n_jobs=n_jobs)
     fit_time = datetime.now()
     for phonetic_block, cluster in clusterer.clusterer.clusterers_.items():
+        if test_signatures_uuids:
+            training_statistics = cluster.score(test_signatures_uuids, test_labels)
```

Comment:
here `cluster` is not an instance of your class (`inspire_disambiguation.core.cluster.ml.models.Cluster`) due to the `for` roop above. I doubt this works like this.

Reply:
Yes, that's true. Can I move this part before/after the for loop and display global statistics (for all the clusters)?

Reply:
yes, I think that would work.